src/driver.rs

```diff
@@ -82,8 +75,21 @@ macro_rules! trace {
 #[macro_export]
 macro_rules! log {
     ($level:ident, $driver:expr, $($args:tt)+) => {
-        if $driver.log_level() >= $crate::driver::LogLevel::$level {
-            $driver.log($crate::driver::LogLevel::$level, format_args!($($args)+));
+        let meta = log::Metadata::builder()
```

Comment:
You should check log::max_level first, since typically enabled shouldn't even be called if the max_level is too low. See https://github.com/rust-lang-nursery/log/blob/master/src/macros.rs#L33-L46

Reply:
Hmm, `log::max_level` is global, though—if we're not using the `log` crate, we don't want to use its max level, since it might be different from our actual logger's. (That's the case on Desktop, at least. `log::max_level()` doesn't know the max log level for Sync logs).

Reply:
I wrapped `log::max_level()` in `Driver::max_log_level()`, so this should work now!